pyproject.toml

```diff
@@ -153,3 +152,10 @@ upload_to_vcs_release = true
 
 [tool.pytest.ini_options]
 timeout = 15
+
+[tool.ruff]
+ignore = ["D1", "E501"]
```

Comment:
why ignore `D1 and `E501` altogether?

Reply:
The `.pylintrc` disables missing-docstrings and line-length issues, so I thought I would replicate the behaviour here. However it would seem that these rules are not needed since all the checks pass.